templates/tpos/tpos.html

```diff
@@ -197,12 +206,23 @@ <h5>
       <q-page-sticky position="top-right" :offset="[18, 162]">
         <q-btn
           @click="atm"
-          v-if="withdrawamtposs > 0"
+          v-if="withdrawamtposs > 0 && !atmMode"
           fab
           icon="atm"
           color="primary"
         ></q-btn>
       </q-page-sticky>
+<!--
```

Comment:
what is this comment?

Reply:
Removal of 2nd button during this PR development. That part of code is removed completely on later commit (https://github.com/lnbits/tpos/pull/68/commits/1366a57f3dd32e25459a8cba8a2c1d5f83c44beb)

Reply:
its not in this pr though, can you remove it here?

Reply:
> its not in this pr though, can you remove it here?
>
My bad. Now better?